tests/system/testdata/connections/create.txtar

```diff
@@ -4,32 +4,28 @@ return code == 0
 output equals 'project_id: prj_00000000000000000000000001'
 
 # Negative test: create connection without name.
-ak connection create --project my_project --integration http --connection-token my_url_path
+ak connection create --project my_project --integration http --connection-token tok
```

Comment:
Please use a human-friendly string like `"my_token"` instead of the meaningless abbreviation `"tok"` (in all instances below too).

Reply:
done